minecraft_servers/phoenix/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "phoenix",
+  "nice_name": "Phoenix",
+  "direct_ip": "rainbowfurry.com",
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://rainbowfurry.com",
+    "web_shop": "https://phoenix-network-store.tebex.io"
+  },
+  "gamemodes": {
+    "clashofclans": {
+      "name": "Clash of Clans",
+      "command": "/quickjoin CoM",
+      "url": "https://www.rainbowfurry.com/ClashOfClans",
```

Comment:
404

Reply:
I also removed the URL for the moment the page went back to maintain but will be added soon...
that's just unnecessary and more duplication of effort later, just because a site is offline at the moment